templates/definition/charger/nrgkick-gen2.yaml

```diff
@@ -0,0 +1,18 @@
+template: nrgkick-gen2
```

Comment:
Please name template type and file same as charger implementation for sake of consistency. Changing either way is fine.